examples/vscode_advanced_debugging/README.md

```diff
@@ -0,0 +1,70 @@
+# Advanced Debugging with VSCode
+
+Contributor: [zhezh](https://github.com/zhezh/BlenderProc)
+
+In this example, we will describe how to debug the python code executed by Blender's python interpreter, e.g.  `src/run.py` which can not be debugged directly in PyCharm or an other IDE. Although, we focus on debugging with the Blender's python interpreter, this method is general.
+
+## Preparation
+- This tutorial is aimed for debugging in: [VSCode](https://code.visualstudio.com/Download)
+- [ptvsd](https://github.com/microsoft/ptvsd), the python package, which we are going to use, the installation is part of the tutorial.
+- [examples/basic](https://github.com/DLR-RM/BlenderProc/tree/master/examples/basic) Make sure you have run this example before.
+
+## Steps
+Firstly, add `ptvsd` into `setup / pip` of `examples/basic/config.yaml`. The first few lines should be like below:
+```
+{
```

Comment:
Can you create an example config file for this please, each example folder should contain a config.yaml.

Reply:
added a config file. 
But this debugging example is based on the specific `basic` example, maybe I need to copy all the files from `basic` example into this folder, and change the description in readme to be independent of the `basic` example.

Reply:
It depends, you could remove the `<args:0>` and `<args:1>`, from the config and replace them with the paths to the example files.

Then it should work as expected, without copying those files.